### PR TITLE
Add missing common-instancetypes prow configuration

### PR DESCRIPTION
**Pull request by lyarwood**

### CODE REVIEW

File: github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml [hunk 1]
```diff
@@ -351,6 +353,10 @@ branch-protection:
           branches:
             main:
               protect: true
+        common-instancetypes:
+          branches:
+            master:
```
Comment (dhiller) [inline github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml:358]: Are you sure that the branch is still called `master`?

**Review by dhiller — CHANGES_REQUESTED** [review]
I think you need to correct the branch name in branch-protection, other that that looks fine to me.

File: github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml (hunk 1 above)
Comment (lyarwood) [reply github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml]: Ah apologies, nice catch!

**Review by dhiller — APPROVED** [review]
/approve

**Pull request merged**